### ISSUE TITLE
About first stage proxy code training

### ISSUE THREAD

**Issue by zichengsaber** [bytedance/1d-tokenizer#14]
hello，I want to ask about the details about the first stage proxy code training. From the beginning training the cross entropy loss will round 6.9 (-log(1/1024)), and after training what the value of CE loss will lead to the results of baseline 
![image](https://github.com/bytedance/1d-tokenizer/assets/49311475/62342ab7-57a2-4dcc-a5d5-78a7e83b5141)

**Comment by MikeWangWZHL** [comment]
Also want to get more details on this; And for the decoder finetuning, are you (1) only finetune VQGAN decoder (2) or finetune the Transformer and VQGAN decoder?

**Comment by cornettoyu** [comment]
Hi,

Thank you all for your interests in our work and patience. We have released the two-stage training code and feel free to check it.
